projects/aca-content/src/lib/components/search/search-results/search-results.component.ts

```diff
@@ -164,7 +174,7 @@ export class SearchResultsComponent extends PageComponent implements OnInit {
 
     this.queryBuilder.configUpdated
       .asObservable()
-      .pipe(takeUntil(this.onDestroy$))
+      .pipe(takeUntilDestroyed(this.destroyRef))
```

Comment:
I think if used in component, takeUntilDestroyed() is enough, without destroyRef as parameter

Reply:
Good catch , added